README.md

```diff
@@ -7,6 +7,7 @@ The repository provides a collection of vision language models, benchmarks, and
 
 ## 💡 News
 
+- [2024/12/04] The arXiv version of VILA-M3 is now available [here](https://arxiv.org/pdf/2411.12915).
```

Comment:
Please link to the arxiv page and not the pdf directly https://arxiv.org/abs/2411.12915

Reply:
Fixed.